img/u/unimelb.edu.au.svg

```diff
@@ -0,0 +1,124 @@
+				<path fill="#FFFFFF" d="M47.442,29.6c-0.089,0.473,0.234,0.935,0.606,1.158l0.068-0.358      C48.063,30.064,47.709,29.754,47.442,29.6z"/>
+				<path fill="#FFFFFF" d="M47.236,30.495c-0.04,0.132-0.157,0.301-0.124,0.447c0.052,0.264,0.469,0.364,0.597,0.757      C47.935,31.336,47.634,30.6,47.236,30.495z"/>
+				<path fill="#FFFFFF" d="M46.939,31.275l-0.183,0.322c-0.304,0.464,0.254,0.829,0.248,1.123c0.169-0.181,0.358-0.664,0.253-0.998      C47.21,31.546,47.103,31.389,46.939,31.275z"/>
+				<path fill="#FFFFFF" d="M45.816,31.346c-0.201,0.079-0.692,0.401-0.942,0.366c0.123,0.156,0.664,0.124,0.857,0.148      c0.337,0.02,0.566-0.362,0.735-0.63L45.816,31.346z"/>
+				<path fill="#FFFFFF" d="M46.349,32.159c-0.664,0.163-0.244,0.943-0.806,1.394c0.386-0.07,0.554-0.075,0.797-0.537      C46.45,32.793,46.501,32.387,46.349,32.159z"/>
+				<path fill="#FFFFFF" d="M41.178,30.682c-0.127,0.507-0.009,0.621,0.232,0.933c-0.117-0.713,0.72-1.01,0.338-1.576      C41.48,30.108,41.246,30.442,41.178,30.682z"/>
+				<path fill="#FFFFFF" d="M43.996,32.646c0.025,0.158,0.657,0.12,0.949-0.008c0.147-0.032,0.641-0.222,0.693-0.4      C44.493,31.977,44.901,32.402,43.996,32.646z"/>
+				<path fill="#FFFFFF" d="M45.346,30.989c0.052,0.15,0.668,0.002,0.935-0.176c0.136-0.059,0.587-0.332,0.609-0.516      C45.716,30.243,46.191,30.588,45.346,30.989z"/>
+				<path fill="#FFFFFF" d="M41.548,29.465c-0.167-0.077-0.636,0.162-0.757,0.249c-0.291,0.133-0.742,0.58-0.64,0.704      C40.935,29.906,40.98,30.492,41.548,29.465z"/>
+			</g>
+			<g>
+				<path fill="#FFFFFF" d="M43.124,27.467c0.397,0.111,0.577,0.347,0.841,0.385c0.31,0.013,0.42-0.079,0.479-0.192      C44.319,27.268,43.528,27.309,43.124,27.467z"/>
+				<path fill="#FFFFFF" d="M44.959,29.226c-0.031-0.413,0.132-0.66,0.077-0.922c-0.092-0.297-0.214-0.37-0.341-0.386      C44.369,28.169,44.673,28.897,44.959,29.226z"/>
+				<path fill="#FFFFFF" d="M43.068,27.802c-0.331-0.117-0.905,0.121-1.093,0.283c0.425-0.042,0.693,0.431,1.199,0.224l0.325-0.171      C43.394,27.97,43.242,27.858,43.068,27.802z"/>
+				<path fill="#FFFFFF" d="M43.722,28.633c-0.106,0.333,0.149,0.901,0.319,1.084c-0.055-0.425,0.409-0.706,0.186-1.207l-0.184-0.32      C43.88,28.301,43.771,28.459,43.722,28.633z"/>
+				<path fill="#FFFFFF" d="M45.38,32.984l-0.368,0.021c-0.542,0.078-0.531,0.621-0.918,0.803c0.244,0.046,0.86-0.045,1.084-0.314      C45.301,33.359,45.375,33.182,45.38,32.984z"/>
+				<path fill="#FFFFFF" d="M41.657,28.511c-0.494,0.177-0.521,0.341-0.646,0.714c0.527-0.494,1.24,0.03,1.494-0.602      C42.3,28.439,41.892,28.434,41.657,28.511z"/>
+				<path fill="#FFFFFF" d="M42.802,29.473c0.044,0.522,0.195,0.592,0.523,0.807c-0.339-0.635,0.352-1.191-0.196-1.597      C42.897,28.833,42.786,29.227,42.802,29.473z"/>
+			</g>
+			<path fill="#FFFFFF" d="M42.192,29.425c-0.027,0.714-0.015,1.262,0.29,1.603c0.194-0.578,0.204-0.839,0.048-1.189     C42.474,29.685,42.336,29.417,42.192,29.425z"/>
+		</g>
+	</g>
+</g>
+</svg>
```

Comment:
I recommend keeping the PNG as it is way smaller than a minified SVG: 639 bytes versus 21 KB.